react/features/recording/components/RecordingLabel.web.js

```diff
@@ -110,10 +116,26 @@ class RecordingLabel extends Component {
                 ? 'with-remote-videos' : 'without-remote-videos'
         ].join(' ');
 
+        let rightIndent = {};
+
+        if (this.props._filmstripVisible
+            && this.props._remoteVideosVisible) {
+            rightIndent
+                = this.props._transcriptionLabelVisible ? '243px' : '200px';
+        } else {
+            rightIndent
+                = this.props._transcriptionLabelVisible ? '123px' : '80px';
```

Comment:
Refactoring to pull out positioning logic could help eliminate this dependency on transcription label. Also, adding inline styles here is inconsistent with how positioning has been handled previously.

Reply:
Agree, this was definitely a try to fix the issue as quickly as possible and then think through a better solution.